trunk-recorder/main.cc

```diff
@@ -1056,6 +1076,10 @@ void handle_message(std::vector<TrunkMessage> messages, System *sys) {
       unit_ack(sys->get_unit_script(), sys->get_short_name(), message.source);
       break;
 
+    case LOCREG:
```

Comment:
I will probably see this defined latter on.... but I am confused what LOCREG stands for.

Reply:
Location Registration Response. I (we?) should maybe use official P25 opcode aliases, in this case `LOC_REG_RESP'

Reply:
I am fine with the shorter version, 'LOCREG'. Can you add in comments for the function and the switch statement?

Reply:
Let's keep this in the PR